### DISCUSSION TITLE
Installing oqtane in Server and 3 remote clients

### DISCUSSION THREAD

**Discussion by engsky** [oqtane/oqtane.framework#2270] [Q&A] [answered]
How installing oqtane framework in reality senario that I have Windows Server with sql server database and three clients remotly dealing with the database that is in the server, so that each client can works with one server database of course without installing the framework in each client with server database connection?

**Comment by sbwalker** [comment, marked as answer]
@engsky are you saying that that you would like to have a single installation of Oqtane which you would like to use for 3 clients - but you would like each client to have their own separate database?

**Reply by engsky** [reply]
@sbwalker  No,  all clients shared one central database. Without installing whole oqrane framework within each client.

**Reply by sbwalker** [reply]
@engsky so are you saying that that you would like to have a single installation of Oqtane with a single database, which you would like to use for 3 clients?

**Reply by engsky** [reply]
@sbwalker yes.

**Reply by engsky** [reply]
@sbwalker clients will have their own database if they are offline but main database in server.

**Reply by sbwalker** [reply]
Oqtane is multi-tenant and can support multiple clients from the same installation. You would install Oqtane on the server and define unique Urls for each client (ie. www.client1.com, www.client2.com, etc...).

**Reply by engsky** [reply]
Thank you a lot @sbwalker . I would like to thank you so much for your strong resources in blazor at all and the information in many subjects.


Do you mean creating sites for each client?  Or urls in iis?
Is it possible to include the user id in the url?  What is the best dealing about this.  Thank you. ♥

**Reply by sbwalker** [reply]
Unique URLs for each client would first need to be configured in IIS. Then in Oqtane you would need to create a separate Site for each client (all Sites can be in the same Tenant/Database). Each Site would be mapped to the URLs defined previously (URLs are known as Aliases in Oqtane). Each Oqtane site has its own distinct set of users, security roles, and content. I would encourage you to watch one of the video tutorials or demos to gain a better understanding.

**Reply by engsky** [reply]
@sbwalker I will do that also watch the videos produced by you. Thank you Mr. Shaun Walker.
